lib/playgroundbook_renderer/page_parser.rb

```diff
@@ -0,0 +1,19 @@
+module Playgroundbook
+  class PageParser
+    def parse_chapter_pages(chapter_contents)
+      page_names = chapter_contents.scan(/\/\/\/\/.*$/).map { |p| p.gsub('////', '').strip }
```

Comment:
my general rule of regexes is to include a comment above for people like me who cannot read them :D